src/theme/Navbar/Logo/index.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import Logo from '@theme/Logo';
+import styles from './styles.module.css';
 export default function NavbarLogo() {
   return (
     <Logo
-      className="navbar__brand"
+      className={'navbar__brand '+ styles.navbar__logo}
```

Comment:
Please don't concatenate class names like that, there is a library for it already added as a dependency (`clsx`).

```suggestion
      className={clsx('navbar__brand ', styles.navbar__logo)}
```